### PR TITLE
PR: merge Rootstock updates on 2020-02-28 

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

**Comment by dhimmel** [comment]
I think this PR has to be merged before GitHub actions will start running. However, once that's the case, we could remove Travis and use GitHub Actions + AppVeyor. @agitter?

**Comment by dhimmel** [comment]
builds for https://github.com/greenelab/deep-review/pull/991/commits/c8221679dbd42ba5ec13d643a3f2364e5904b8cc seem to be timing out. Not sure why.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![HTML Manuscript](https://img.shields.io/badge/manuscript-HTML-blue.svg)](https://greenelab.github.io/deep-review/)
 [![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://greenelab.github.io/deep-review/manuscript.pdf)
-[![Build Status](https://travis-ci.org/greenelab/deep-review.svg?branch=master)](https://travis-ci.org/greenelab/deep-review)
+[![GitHub Actions Status](https://github.com/greenelab/deep-review/workflows/Manubot/badge.svg)](https://github.com/greenelab/deep-review/actions)
+[![Travis Build Status](https://travis-ci.org/greenelab/deep-review.svg?branch=master)](https://travis-ci.org/greenelab/deep-review)
```
Comment (agitter) [inline README.md]: We can remove this if we switch to GitHub Actions+AppVeyor

**Comment by dhimmel** [comment]
> I wonder why the AppVeyor builds still work despite the Travis CI builds timing out?

Travis times out after 10 minutes of inactivity in terms of log output. AppVeyor must be more patient.

**Comment by dhimmel** [comment]
Okay I think the timeout was caused by the datacite DOI resolver being a bit slower than what we used previously combined with having to regenerate the cache for all DOIs. @agitter I think we can merge with the slowness.

One thing we should probably do is freeze the references.

File: content/90.back-matter.md [hunk 2]
```diff
@@ -2,3 +2,344 @@
...
+[@tag:Glorot2011_domain]: url:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.231.3442
+[@tag:Goldsborough2017_cytogan]: doi:10.1101/227645
+[@tag:Gomezb2016_automatic]: arxiv:1610.02415v1
+[@tag:Graphlab]: doi:10.14778/2212351.2212354
+[@tag:Groop1986_islet]: doi:10.2337/diab.35.2.237
+[@tag:Gultepe2014_sepsis]: doi:10.1136/amiajnl-2013-001815
+[@tag:Gulshan2016_dt]: doi:10.1001/jama.2016.17216
+[@tag:Gupta2015_exprs_yeast]: doi:10.1101/031906
+[@tag:Gupta2015_prec]: arxiv:1502.02551
+[@tag:Guetterman]: url:http://www.fasebj.org/content/30/1_Supplement/406.3
```
Comment (agitter) [inline content/90.back-matter.md]: If we freeze references post-merge, we need to add a manual reference for this first.  We're adding a broken DOI link https://doi.org/10.1096/fasebj.30.1_supplement.406.3

**Pull request merged**